katello.gemspec

```diff
@@ -50,6 +50,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_file_client", ">= 0.3.0", "< 0.4.0"
   gem.add_dependency "pulp_ansible_client", "> 0.1", "< 0.3.0"
   gem.add_dependency "pulp_container_client", ">= 1.3.0", "< 1.4.0"
+  gem.add_dependency "pulp_deb_client", "2.3.0b1"
```

Comment:
can you make this a little more generic?    ">= 2.3", "< 2.4" ?

Reply:
Have to check that.
`< 2.4` will still give us 2.4 pre-release versions like `2.4.0b1.dev01591032487`